src/Apps/Home/Components/HomeTrendingArtistsRail.tsx

```diff
@@ -96,7 +96,7 @@ export const HomeTrendingArtistsRailFragmentContainer = createFragmentContainer(
   {
     viewer: graphql`
       fragment HomeTrendingArtistsRail_viewer on Viewer {
-        artistsConnection(sort: TRENDING_DESC, first: 99) {
+        curatedTrendingArtists(first: 99) {
```

Comment:
`99` 🧐 wonder what that was/is about!

Reply:
Actually yeah now that I'm pausing long enough to actually read that argument I'm not sure what it's doing haha. Any thoughts @dzucconi @damassi ?

Reply:
I have no idea why this is set to fetch a crazy large number (why 99 and not 100 too?) but can we actually change this to something sensible like 20? It's been on my mental list for a while; looks ridiculous on the homepage.

Reply:
Yeah I'm happy to change it - holler if anybody knows of a good reason not to!!!

Reply:
I was assuming the original assumption was that there was some set number of them and wanted to fetch them all.